tools/separate_asm.py

```diff
@@ -72,7 +72,7 @@ def find_asm_block(asm_js):
   # closure compiler removes |var Module|, we need to find the closured name
   # seek a pattern like (e.ENVIRONMENT), which is in the shell.js if-cascade for the ENVIRONMENT override
   import re
-  m = re.search(r'(\w+)\s*=\s*"__EMSCRIPTEN_PRIVATE_MODULE_EXPORT_NAME_SUBSTITUTION__"', everything)
+  m = re.search(r'(\w+)\s*=\s*{"__EMSCRIPTEN_PRIVATE_MODULE_EXPORT_NAME_SUBSTITUTION__":1}', everything)
```

Comment:
I would have thought that closure would run at the end after any substitutions.  Is there a reason that it runs before this one?

Reply:
This is used to let closure minify references to `Module`, but to keep one unminified reference around to import an unminified reference to the minified scope.